### PR TITLE
Add pagerduty v2 documentation

### PR DESCRIPTION
**Pull request by mcorbin**

### CODE REVIEW

File: howto.html [hunk 1]
```diff
@@ -2706,7 +2706,10 @@ <h3>Notify with Pagerduty</h3>
 
     <p>Create a <a href="http://riemann.io/api/riemann.pagerduty.html#var-pagerduty">client</a> with your Pagerduty service key, then use <code>:trigger</code> and <code>:resolve</code> to open and close issues for a given host and service.</p>
 
-    <p>You can also pass http options using the <code>:options</code> key.</p>
+    <p>You can also pass http options using the <code>:options</code> key, and a custom formatter for the Pagerduty event with the <code>:formatter</code> key.</p>
+
+    <p>You can use the <a href="https://v2.developer.pagerduty.com/docs/send-an-event-events-api-v2">Pagerduty v2 API</a> by setting the <code>:version</code> key to <code>:v2</code>. In v2, each event can contains a <code>:dedup-key</code> key to handle alert de-duplication.
+</p>
   </div>
```
[the remark this answers is not in the document]
Comment (mcorbin) [reply howto.html]: fixed, thanks ;)

**Comment by mcorbin** [comment]
Codacy seems broken but the error is not very explicit :D

**Comment by jamtur01** [comment]
Yeah. Ignore it. I am on my phone so I can't override. I'll do it tonight.

**Comment by mcorbin** [comment]
Ok ;)

**Pull request merged**